storagenode/piecestore/endpoint_test.go

```diff
@@ -459,6 +381,65 @@ func TestDelete(t *testing.T) {
 	}
 }
 
+func TestDeletePiece(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	planet, err := testplanet.New(t, 1, 1, 1)
+	require.NoError(t, err)
+	defer ctx.Check(planet.Shutdown)
+
+	planet.Start(ctx)
+
+	var (
+		planetSat = planet.Satellites[0]
+		planetSN  = planet.StorageNodes[0]
+	)
+
+	var client *piecestore.Client
+	{
+		dossier, err := planetSat.Overlay.DB.Get(ctx.Context, planetSN.ID())
+		require.NoError(t, err)
+
+		client, err = piecestore.Dial(
+			ctx.Context, planetSat.Dialer, &dossier.Node, zaptest.NewLogger(t), piecestore.Config{},
+		)
+		require.NoError(t, err)
+	}
+
+	t.Run("Ok", func(t *testing.T) {
+		pieceID := storj.PieceID{1}
+		data, _, _ := uploadPiece(t, ctx, pieceID, planetSN, planet.Uplinks[0], planetSat)
+		require.NoError(t, err)
```

Comment:
This require.NoError should be removed I think

Reply:
Done